credential-token_polygon/contracts/WIW3CredentialToken.sol

```diff
@@ -0,0 +1,295 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
```

Comment:
Can we please follow this framework for naming files? e.g. `credential-token_goerli` changes to `credential-token-goerli`